perfkitbenchmarker/benchmarks/cloudsuite_websearch_benchmark.py

```diff
@@ -150,13 +140,13 @@ def _BuildIndex(solr_nodes, fw):
   def DownloadIndex(vm):
     solr_core_dir = posixpath.join(vm.GetScratchDir(), 'solr_cores')
     vm.RobustRemoteCommand('cd {0} && '
-                           'wget {1} && '
-                           'tar zxvf index -C {2}'.format(
+                           'curl -L {1} | '
```

Comment:
Do we need to switch to vm.Install('curl') in Prepare function instead of 'wget' now? Otherwise looks good to me too.